test/concourse/model/traits/filter_test.py

```diff
@@ -0,0 +1,140 @@
+import dataclasses
+import typing
+
+import pytest
+
+import concourse.model.traits.filter as examinee
+
+
+# Dummy dataclass for testing
+@dataclasses.dataclass
+class Dummy:
+    name: str
+
+
+@pytest.fixture
+def dummy():
+
+    def _dummy(
+        name: str = 'Dummy',
+    ):
+        return Dummy(
+            name=name,
+        )
+
+    return _dummy
+
+
+class TestMatchingFilter:
+    @pytest.fixture
+    def rule(self):
```

Comment:
why define the first fixture (which we might not actually need..) on module-level, and the second one as a method?